_posts/2021-03-05-scala-3-developers-preview-survey-results.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: blog-detail
+post-type: blog
+by: Scala Center
+title: Scala 3 Developer’s Preview Survey Results
+---
+## Context
+
+After 8 years of development, the Dotty project is well on its way to become the new Scala compiler,
+known in the community as Scala 3.
+
+Since announcing this news (ScalaDays Berlin 2018), it has taken a good 2+ years to finalize the changes,
+sync the community’s efforts, publish migration paths, and create a new ecosystem of tools and libraries.
+We are very near to being able to take the final step - releasing Scala 3.0.
+We are still not there yet but are closer than ever.
+
+In December 2020 Scala 3.0.0-M3 was released, the last milestone before the RC1 release, scheduled for January 2021.
+
```

Comment:
I will give my usual feedback here that I give on everybody's posts :-)

The blog post should immediately say what it's about, right at the start. Don't keep the user waiting, let them know what is coming so they can decide whether to keep reading.

The above three paragraphs are an unnecessary preamble. If this recap is actually important to include, let it come second or third. The blog post should begin:

> In December, we published a survey sharing the status of the Scala 3 release and asking the Scala community to give us their opinions, concerns and advice. Today, we are pleased to share what we learned with you.

Reply:
You might also consider sharing highlights of the results right at the top. "Here are a few of the most important things we learned: ..." and then pick 3 things or so. (But if suitable candidates for that don't jump out at you, then never mind, I guess.)